cluster/images/etcd/Makefile

```diff
@@ -26,16 +26,21 @@
 # Except from etcd-$(tag) and etcdctl-$(tag) binaries, we also
 # need etcd and etcdctl binaries for backward compatibility reasons.
 # That binary will be set to the last tag from $(TAGS).
+
+include ../../../hack/make-rules/Makefile.manifest
+
 TAGS?=2.2.1 2.3.7 3.0.17 3.1.12
-REGISTRY_TAG?=3.1.12
+REGISTRY_TAG?=3.2.12
```

Comment:
I was not expecting the version change here. Is there a reason we are doing that?

Reply:
lot of changes went in the latest etcd makefile, lemme rebase first.